NodeEdge/Dockerfile

```diff
@@ -46,7 +44,10 @@ RUN if [ -z "$EDGE_DRIVER_VERSION" ]; \
   && rm /tmp/msedgedriver_linux64.zip \
   && mv /opt/selenium/msedgedriver /opt/selenium/msedgedriver-$EDGE_DRIVER_VERSION \
   && chmod 755 /opt/selenium/msedgedriver-$EDGE_DRIVER_VERSION \
-  && sudo ln -fs /opt/selenium/msedgedriver-$EDGE_DRIVER_VERSION /usr/bin/msedgedriver
+  && ln -fs /opt/selenium/msedgedriver-$EDGE_DRIVER_VERSION /usr/bin/msedgedriver
+ENV SE_DRIVER_OPTS="-Dwebdriver.edge.driver="${EDGE_DRIVER_PATH_PROPERTY:-"/usr/bin/msedgedriver"}" "$SE_DRIVER_OPTS
```

Comment:
This won't be used as we are always setting the browser driver before launching the Grid.